sql/core/src/test/resources/sql-tests/results/group-by-ordinal.sql.out

```diff
@@ -122,7 +122,7 @@ select a, b, sum(b) from data group by 3
 struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
-aggregate functions are not allowed in GROUP BY, but found sum(data.b)
+GROUP BY expression in position 3 is or contains an aggregate function. Aggregate functions are not allowed in GROUP BY, but got sum(data.b) AS `sum(b)`; line 1 pos 39
```

Comment:
I'm reading the error message again but feel it's hard to understand. How about
```
GROUP BY 3 refers to an expression that is or contains an aggregate function. Aggregate functions are not allowed ...
```

Reply:
Done